lib/screens/v2/candidate_generator/widgets/reconstructed_alert.ex

```diff
@@ -74,4 +78,18 @@ defmodule Screens.V2.CandidateGenerator.Widgets.ReconstructedAlert do
           false
       end
   end
+
+  defp get_stations(alert) do
+    %{alert: alert}
+    |> BaseAlert.informed_entities()
+    |> Enum.map(fn %{stop: stop_id} -> stop_id end)
+    |> Enum.filter(&String.starts_with?(&1, "place-"))
+    |> Enum.uniq()
+    |> Enum.map(
+      case &Stop.fetch_stop_name(&1) do
```

Comment:
You'll need to move the `&` to the front of `case` for this to work as expected!
```suggestion
      &case Stop.fetch_stop_name(&1) do
```

Currently the way this will evaluate is:
- `case` gets passed the anonymous function `&Stop.fetch_stop_name(&1)`, which will always match `name -> name`.
- The whole case expression then evaluates to `&Stop.fetch_stop_name(&1)`, which then gets passed as the callback to `Enum.map`. `fetch_stop_name` runs on each element of the list without any error handling.

This will still ~kind of work, (until there's an API error) but the `case` expression isn't actually doing any error handling since it gets passed the function, not the result of the function's evaluation.